test/fit_tests/deploy/onrack_stack_init.py

```diff
@@ -0,0 +1,306 @@
+        # and active workflows are all either present or complete. If  the returned is false,
+        # there was a timeout and all nodes have not obtained obmSetting, SKUs, or all active
+        # workflows have not completed.
+        # Wait 10 minutes ( MAX_CYCLES * 10 seconds) for this to occur.
+        self.assertTrue(self.check_for_active_workflows(MAX_CYCLES), "Node discovery not completed")
+        
+    def check_for_active_workflows(self, max_time):
+        '''
+        Determine if are any active workflows.
+        :param    Time to wait (in 10 second intervals)
+        :return:  True  - No active workflows
+                  False - Workflows are active
+        '''
+        for dummy in range(0, max_time):
+            nodes_data = fit_common.rackhdapi("/api/2.0/nodes")
+            if nodes_data['status'] == 200 and len(nodes_data['json']) > 0:
+                # if there are nodes present, determine if discovery has completed on them
+                discovery_complete = True
+
+                for node in nodes_data['json']:
+                    if node['type'] == 'compute':
+                        self.assertIn('id', node, 'node does not contain id')
+                        node_id = node['id']
+                        # determine if there are any active worlflows. If so, discovery not completed
+                        awf_data = fit_common.rackhdapi('/api/1.1/nodes/'
+                                                           + node_id
+                                                           + '/workflows/active')
+                        # 204 indicates "No Content"
+                        if awf_data['status'] != 204 or 'sku' not in node:
+                            discovery_complete = False
+                            break
+
+                if discovery_complete:
+                    return True
+                fit_common.time.sleep(10)
+        return False
+
+    def test08_install_obm_credentials(self):
+        print "**** Install OBM credentials."
+        # install OBM credentials via workflows
+        count = 0
+        for creds in fit_common.GLOBAL_CONFIG['credentials']['bmc']:
+            # greate graph for setting OBM credentials
+            payload = \
+            {
+                "friendlyName": "IPMI" + str(count),
+                "injectableName": 'Graph.Obm.Ipmi.CreateSettings' + str(count),
+                "options": {
+                    "obm-ipmi-task":{
+                        "user": creds["username"],
+                        "password": creds["password"]
+                    }
+                },
+                "tasks": [
+                    {
+                        "label": "obm-ipmi-task",
+                        "taskName": "Task.Obm.Ipmi.CreateSettings"
+                    }
+            ]
+            }
+            api_data = fit_common.rackhdapi("/api/1.1/workflows", action="put", payload=payload)
```

Comment:
I see these test code sometimes use API 1.1, sometimes use API 2.0.  Need to use the same API version.